src/client/src/lib.rs

```diff
@@ -86,7 +86,7 @@ mod error;
 mod client;
 mod admin;
 mod params;
-mod consumer;
+pub mod consumer;
```

Comment:
Don't think we want to expose entire module yet.  @nicholastmosher concur?
Let's just expose what is need for this change